actions/sequester/Quest2GitHub/AzDoClientServices/JsonPatchDocument.cs

```diff
@@ -69,6 +69,18 @@ public class JsonPatchDocument
     public object? Value { get; init; }
 }
 
+public class Relation
+{
+    [JsonPropertyName("rel")]
+    public string RelationName { get; init; } = "ArtifactLink";
+
+    [JsonPropertyName("url")]
+    public required string Url { get; init; }
+
+    [JsonPropertyName("attributes")]
+    public IDictionary<string, object> Attributes{ get; } = new Dictionary<string, object>();
+
```

Comment:
```suggestion
    public IDictionary<string, object> Attributes{ get; } = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
```

You may want to consider making this case insensitive.